smriprep/workflows/anatomical.py

```diff
@@ -410,10 +421,10 @@ def _check_img(img):
             (('outputnode.bias_corrected', _pop), 'inputnode.moving_image')]),
         (buffernode, anat_norm_wf, [('t1w_mask', 'inputnode.moving_mask')]),
         (anat_norm_wf, outputnode, [
-            ('poutputnode.standardized', 'std_preproc'),
-            ('poutputnode.std_mask', 'std_mask'),
-            ('poutputnode.std_dseg', 'std_dseg'),
-            ('poutputnode.std_tpms', 'std_tpms'),
+            ('outputnode.standardized', 'std_preproc'),
+            ('outputnode.std_mask', 'std_mask'),
+            ('outputnode.std_dseg', 'std_dseg'),
+            ('outputnode.std_tpms', 'std_tpms'),
```

Comment:
This is going to break existing FreeSurfer. We'll need to revert this. Was it breaking something?